### PR TITLE
[deployments-k8s#3053] Increase CPU limits for NSM applications

### PR DESCRIPTION
**Pull request by Bolodya1997**
## Description
Increases CPU limits for NSM applications. Initial Request for NSE Composition example starts taking less than `10s`.

## Issue
https://github.com/networkservicemesh/deployments-k8s/issues/3053

## How Has This Been Tested?

- [ ] Added unit testing to cover
- [x] Tested manually
- [ ] Tested by integration testing
- [ ] Have not tested

## Types of changes

- [x] Bug fix
- [ ] New functionality
- [ ] Documentation
- [ ] Refactoring
- [ ] CI

### CODE REVIEW

File: examples/features/nse-composition/README.md [hunk 1]
```diff
@@ -71,8 +71,6 @@ spec:
           env:
             - name: NSM_NETWORK_SERVICES
               value: kernel://nse-composition/nsm-1
-            - name: NSM_REQUEST_TIMEOUT
-              value: 75s
```
Comment (denis-tingaikin) [inline examples/features/nse-composition/README.md:75]: Should we increase CPU limit for nsmgr? As I remember you said this is required.

Comment (denis-tingaikin) [reply examples/features/nse-composition/README.md:75]: Ah I see you did it

File: apps/nsmgr/nsmgr.yaml [hunk 2]
```diff
@@ -51,7 +51,7 @@ spec:
           resources:
             limits:
               memory: 100Mi
-              cpu: 200m
+              cpu: 400m
```
Comment (denis-tingaikin) [inline apps/nsmgr/nsmgr.yaml:54]: Could you please look at https://github.com/networkservicemesh/deployments-k8s/pull/3050?

Comment (Bolodya1997) [reply apps/nsmgr/nsmgr.yaml:56]: Looks like we need either to merge #3050 first, or just replace it with this PR.
Updated this PR.

**Pull request merged**